api/apps/v1alpha1/nimcache_types.go

```diff
@@ -110,13 +110,17 @@ type GPUSpec struct {
 
 // DataStoreSource references a model stored on NVIDIA DataStore service.
 type DataStoreSource struct {
-	// The endpoint for datastore
+	// HF-compatible datastore endpoint
+	// +kubebuilder:validation:Pattern=`^http(s)?://.*v1/hf/?$`
```

Comment:
we can remove the validation with `v1/hf` suffix here so that we can leverage this to pull models from HF using huggingface-cli for e.g.